install.sh

```diff
@@ -260,6 +260,19 @@ execute() {
   fi
 }
 
+retry() {
+  local tries="$1" n="$1"
+  shift
+  while [[ $((n--)) -gt 0 ]]
+  do
+    if "$@"
+    then
+      return
+    fi
+  done
```

Comment:
May want to do exponential back-off here.

Reply:
Good idea, done.